lib/checks/color/link-in-text-block-style-evaluate.js

```diff
@@ -26,18 +28,22 @@ export default function linkInTextBlockStyleEvaluate(node) {
 
   this.relatedNodes([parentBlock]);
 
-  if (elementIsDistinct(node, parentBlock)) {
-    return true;
-  }
-  if (hasPseudoContent(node)) {
-    this.data({ messageKey: 'pseudoContent' });
-    return undefined;
+  for (const inlineNode of inlineNodes) {
+    if (elementIsDistinct(inlineNode, parentBlock)) {
+      return true;
+    }
+
+    if (hasPseudoContent(inlineNode)) {
```

Comment:
If one `inlineNode` passes `elementIsDistinct` and a different one doesn't but `hasPseudoContent`, should this check return `true` or `undefined`? In this implementation, the answer depends on which order we analyze the `inlineNodes` in, which doesn't seem right to me; I think it would probably make more sense to do one loop for the `elementIsDistinct` checks, pass if any `inlineNode` is distinct, and only then do a second loop for the `hasPseudoContent` checks.